tests/sample-project.yaml

```diff
@@ -45,7 +45,7 @@ dev:
           Protocol: TCP
     - name: test-dev-internal
       hosted_zone: kyrtest.pf.dsd.io.
-      scheme: internet-facing
+      scheme: internal
```

Comment:
I tried to get this to work before (internal), but for some reason it wouldn't - does it now?